compiler-rt/lib/radsan/radsan_context.cpp

```diff
@@ -43,11 +44,12 @@ void Context::bypassPush() { bypass_depth_++; }
 void Context::bypassPop() { bypass_depth_--; }
 
 void Context::expectNotRealtime(const char *intercepted_function_name) {
+  ENSURE_RADSAN_INITED();
   if (inRealtimeContext() && !isBypassed()) {
     bypassPush();
     printDiagnostics(intercepted_function_name);
     if (get_error_action_() == OnErrorAction::ExitWithFailure) {
-      exit(EXIT_FAILURE);
+      exit(common_flags()->exitcode);
```

Comment:
Cool! Configurable exitcode for free, just like the other sanitizers

Reply:
I agree this is cool... but let's make sure we have a compelling use case for it before we take on the maintenance 👍

Reply:
Makes sense! Happy to talk more about this on our eventual call